pkg/mod/automod_test.go

```diff
@@ -307,7 +308,8 @@ func comparePlayerNotorieties(pnrs []*ms.NotorietyReport, ustore pkguser.Store,
 		}
 		for gameIndex := range pnrs[idx].Games {
 			ge := pnrs[idx].Games[gameIndex]
-			ga := games[gameIndex]
+			// The games are retrieved in descending chronological order
+			ga := games[(len(games)-1)-gameIndex]
```

Comment:
is this tested? although the db returns the first 5 in descending chronological order, `GetNotoriousGames` then populates the `items` in reverse order, so it should still return them in chronological order (last 5 of ABCDEFG is CDEFG not GFEDC)?

Reply:
Oof, this passed the tests, but by a total fluke. Since time.Unix is in seconds, many of the timestamps for the notorious games are identical. I have no idea how the tests managed to pass, but I need to fix this. What do you think the best way is? I thought of these workarounds but I don't really like any of them:

- Replace the `time.Now().Unix()` call with some mod function that returns an incrementing value for testing and the actual unix timestamp otherwise (sad).
- Add a one second delay in the `playGame` helper function (very sad).
- Don't even check the order since it's all aesthetic anyway (also quite sad).
- Migrate all existing notorious games to some more granular time measurement and start using that time measurement (ugh).

The first one is maybe the least sad and the last one is maybe the most correct, but there could stuff I haven't thought of.